interface/billing/sl_eob_search.php

```diff
@@ -33,6 +33,14 @@
 require_once("$srcdir/sl_eob.inc.php");
 require_once("$srcdir/formatting.inc.php");
 require_once("$srcdir/classes/class.ezpdf.php");//for the purpose of pdf creation
+require_once("$srcdir/api.inc");
+require_once("$srcdir/forms.inc");
+require_once("$srcdir/../controllers/C_Document.class.php");
```

Comment:
why were all these included?  I see nothing in this code that used document class etc ...